ndsi/formatter.py

```diff
@@ -188,7 +188,9 @@ def encode_msg(self, value: GazeValue) -> DataMessage:
 
 class _GazeDataFormatter_V3(GazeDataFormatter):
     def decode_msg(self, data_msg: DataMessage) -> GazeValue:
-        raise NotImplementedError()  # FIXME
+        ts, = struct.unpack("<d", data_msg.header)
+        x, y = struct.unpack("<ff", data_msg.body)
+        return GazeValue(x=x, y=y, timestamp=ts)
```

Comment:
I've got this implementation from here (`v3`):
https://github.com/pupil-labs/pyndsi/blob/master/ndsi/sensor.pyx#L282-L285
so I assumed it's working/needed.

I've updated the implementation to drop support for gaze data in `v3`. I think 'ValueError' makes more sense, since `NotImplementedError` implies that it _will_ be implemented at some point in the future.